Reflection probes

dinomut1 opened the pull request:
Scratchboard for getting reflection probes working. Contains some antipatterns that will be resolved before marking ready for review

[Edit: Linking the Confluence doc]

https://tsu.atlassian.net/wiki/spaces/IREE/pages/2305720326/Reflection+Probes+Spatial+Skyboxes

dinomut1 committed:
mockup

dinomut1 committed:
add editor ui

dinomut1 committed:
invert distance weighting

dinomut1 committed:
add to component shelf

Rezmason reviewed (approved):
Looks good though I'd love to see it in action

dinomut1 committed:
checkpoint

dinomut1 committed:
combine reflection probe system and envmap component
add "Bake Probes" button
add resource management and proper cleanup
add single dedicated webgl renderer

dinomut1 committed:
localize "bake probes" button

dinomut1 committed:
localize "bake probes" button
